library/psa_crypto_ecp.c

```diff
@@ -76,31 +75,28 @@ psa_status_t mbedtls_psa_ecp_load_representation(
          * format, meaning their curve_bytes is equal to the amount of input. */
     }
 
-    if( explicit_bits )
-    {
+    if (explicit_bits) {
         /* With an explicit bit-size, the data must have the matching length. */
-        if( curve_bytes != PSA_BITS_TO_BYTES( curve_bits ) )
-            return( PSA_ERROR_INVALID_ARGUMENT );
-    }
-    else
+        if (curve_bytes != PSA_BITS_TO_BYTES(curve_bits))
+            return(PSA_ERROR_INVALID_ARGUMENT);
+    } else
     {
         /* We need to infer the bit-size from the data. Since the only
-         * information we have is the length in bytes, the value of curve_bits
-         * at this stage is rounded up to the nearest multiple of 8. */
-        curve_bits = PSA_BYTES_TO_BITS( curve_bytes );
+        * information we have is the length in bytes, the value of curve_bits
+        * at this stage is rounded up to the nearest multiple of 8. */
```

Comment:
Weird: some comment continuation lines have the `*` indented to the same level as the opening `/`, rather than the opening `*`. Only some, and I can't see what's different about them.

Reply:
This is something I've noticed, but I've not investigated it yet. It could be related to
```c
/*
 * Multi
 * Line
 * Comment
 */
 ```
 vs
 ```c
 /* Multi
  * Line
  * Comment */

Reply:
Indeed, it seems to happen only in that case. But not always. It's not limited to places where uncrustify is already reindenting. I haven't found a place where uncrustify did reindent the comment for other reasons and got it right, but maybe there are such places and I just didn't find them.

Reply:
I haven't seen a correct indent for the second (open-close-same-line) style, but there are examples like [this one](https://github.com/Mbed-TLS/mbedtls/pull/6397/files#r998000524) for the open-close-separate-line style.

Reply:
Actually there is also the comment on [this line](https://github.com/davidhorstmann-arm/mbedtls/commit/4db6df7b40b162ef6c8a77fd329e90ee95f993db#r87132188), which was corrected correctly even though it has the open-close-same-line style.

(Alternate, [non-diff link](https://github.com/davidhorstmann-arm/mbedtls/blob/uncrustify-experimentation-4/library/ecp_curves.c#L5031) as GitHub sometimes can't render large diffs).

Reply:
There are plenty of places where the first line has text and the continuation has the star aligned with the star. What I haven't checked is if any of those places have been reindented.
```
ag '^( +)/\*.+\n\1 \*' library/*.c
```
(`apt-get install silversearcher` or `brew install the_silver_searcher` or https://github.com/ggreer/the_silver_searcher)

Reply:
The places I've listed are all where uncrustify has changed the continuation line's alignment, e.g. the following:
```diff
     /* Add |c| * 2^bits to the absolute value. Since c and N are
-    * negative, this adds c * 2^bits. */
+     * negative, this adds c * 2^bits. */
     mbedtls_mpi_uint msw = (mbedtls_mpi_uint) -c;
```
Looking at these does not give me much confidence in the consistency of uncrustify, but I'll keep playing with the config.